open_samus_returns_rando/files/levels/s040_area4.lua

```diff
@@ -649,5 +649,12 @@ function s040_area4.OnExit_ChangeCamera_023()
   Game.SetCollisionCameraLocked("collision_camera_023_B", false)
 end
 function s040_area4.OnSubAreaChange(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_, _ARG_4_)
+  -- delay activation of this SG because it throws memory errors if you stay at the
+  -- area where the gamma metroid dies
+  if _ARG_0_ ~= "collision_camera_013" and Scenario.ReadFromBlackboard("Arena_Gamma_001_AllDead", false) then
+    if Game.GetEntity("SpawnGroup019") ~= nil then
+      Game.GetEntity("SpawnGroup019").SPAWNGROUP:EnableSpawnGroup()
+    end
+  end
```

Comment:
Here is the part we should verify on citra and console that it works without including sg 20 and sg 21